README.md

```diff
@@ -1,2 +1,85 @@
+
+### Prerequisites
+
+- [docker CE][docker-install] (Community Edition)
+- [docker-compose][docker-compose-install]
+- git
+- make
+
+
+## Running the Software
+The first step is to clone this repository. The following commands are defined in a Makefile, and can be run as follows:
+
+```
+make up
+```
+This command will create the mysql and metabase containers and show the logs for each
+
+```
+make up-detach
+```
+This command will create the mysql and metabase containers in detached mode (no logs).
+
+```
+make down
+```
+This command will stop and remove the mysql and metabase containers. The mysql volume containing the sql backups will
+persist locally however, so only make up or make up-detach is needed to start the software again.
+
+#### Restoring Database Backups
+Once the `make up` command has completed (about 2-3 minutes), open a new terminal tab and run `bin/restore-database-backups.sh`. It will restore databases from all .sql files in the folder ./database-backups. You may need to run `chmod +x ./bin/restore-database-backups.sh`, to make the file executable.
+
+### Using and Configuring Metabase
+
+If you are using a backup from our current deployment of edu-nexted, the database name in the sql backups
+will be 'mattermost_test'. This is what Metabase is currently configured to look for, so it will run automatically with this
+configuration. If you are restoring an sql backup which contains databases other than 'mattermost_test', see the section
+'Setting Up New Databases in Metabase' below.
+
+## View the Dashboard
+
+Once you navigate to http://localhost:3000, you will be shown the metabase sign-in promt. Ask a developer for the login credentials. Once logged in, you will be presented with the metabase homepage. I have pinned the Mattermost dashboard to the
+top, under the heading 'Start Here'. If you click on it, you will be taken to the dashboard, which shows all of the
+questions, and a brief summary of each.
+
+## View a Question and Download Reports
+
+If you click on a question in the dashboard, you will be taken to the query builder, which also shows a larger
+visualisation of the data. You can change the way the data is visualised using the dropdown under 'Visualisation. You can
+edit/view the sql query by clicking 'OPEN EDITOR'. Some questions have optional parameters, for which values can
+be provided in the text fields above the data visualisation. After running the query, you can click the down arrow on the top right of the data visualisation to download the data in various formats.
+
+You can run the same questions on different databases by clicking 'OPEN EDITOR' in the question, and using
+the dropdown to select a different database. See 'Setting Up New Databases in Metabase' below.
+
+## Making Changes in Metabase
+
+If you make any changes to a question or dashboard, remember to click SAVE. At this point, you will want to commit the
```

Comment:
I haven't looked into how one would add an image to a README.md. Do you know how @jaedoucette or @brecriffs ?

Reply:
![Image description](link-to-image)

Reply:
Doh! `![Image description](link-to-image)`

Reply:
yeah but where would that image live, and how do you get it there? Ah you're suggesting committing the image perhaps in a docs folder or `docs/images` folder so the `link-to-image` would be `docs/images/image-name.jpg` or some such.

Reply:
Turns out the README supports ```<img src="docs/images/question-save-btn-preview.png" width="500"/>```. Added this.

Committed locally so it is not in the .gitignore. Would it be better to have this /docs folder only in the remote repo?

Reply:
Also, is it desirable to have screenshots accompanying every instance that the README says 'click this'?

Reply:
I tend to think so, but it depends on the hassle to get them and target audience. Here, I think the long run goal is for less technical people to be able to run this from the docs, in which case more pictures seems good.

Reply:
> Committed locally so it is not in the .gitignore. Would it be better to have this /docs folder only in the remote repo?

@brecriffs I'm not sure what you mean here.

Also width=500? Does the syntax that John mentioned not work well for your image? (sorta makes me wonder how big an image you created)

Reply:
I just meant that the added `/docs/images` folder was added locally, then committed, and I didn't add it to any gitignore. I guess I was wondering if you think it's desirable to have the /docs folder tracked? My thinking is, it might be nice to have the ability to add/edit some text/images in that folder locally.

Reply:
I just liked the syntax of the img tag I guess. The ability to specify width seems nice depending on the image. 500px looked nice in the docs on my laptop so I went with that.

Reply:
If the images aren't committed and they are referenced by the readme (or other docs), how will those images be found to be displayed?
Certainly you can put files in that directory and not commit them without them being in the gitignore, it just can get tiresome seeing them listed as untracked all the time. You shouldn't be blindly committing every file that looks new or changed (using `git add` w/o specifying what to add), you should be committing files deliberately.

So what files are you thinking you'd want in docs/images locally but not committed? I can't think of any.

Reply:
I guess my thought was, there might not be any sense in have the /docs/images folder locally, so if there was a way to have that folder only in the remote branch, then that could be desirable.